src/Symfony/Bundle/FrameworkBundle/Tests/Functional/NotificationTest.php

```diff
@@ -11,13 +11,21 @@
 
 namespace Symfony\Bundle\FrameworkBundle\Tests\Functional;
 
+use Symfony\Bridge\PhpUnit\ExpectDeprecationTrait;
+
 final class NotificationTest extends AbstractWebTestCase
 {
+    use ExpectDeprecationTrait;
+
     /**
      * @requires function \Symfony\Bundle\MercureBundle\MercureBundle::build
+     *
+     * @group legacy
      */
     public function testNotifierAssertion()
     {
+        $this->expectDeprecation('Since symfony/framework-bundle 6.3: The "notifier.logger_notification_listener" service is deprecated, use "notifier.notification_logger_listener" instead.');
```

Comment:
A legacy test is one that should be removed in 7.0
Since that's not the case here, it means something is missing to *not* trigger the deprecation.

Reply:
I was unable to determine how to not trigger the depreciation, some help would be appreciated 😕!